pkg/dashboard/distro/.gitignore

```diff
@@ -0,0 +1,2 @@
+!/default_distro_info.go
```

Comment:
No need for this line?

Reply:
There's a `default*` rule in root/.gitignore.